specification/purview/data-plane/Azure.Analytics.Purview.MetadataPolicies/stable/2021-07-01/purviewMetadataPolicy.json

```diff
@@ -0,0 +1,535 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview Metadata Policies Service REST API Document",
+    "version": "2021-07-01"
+  },
+  "paths": {
+    "/metadataRoles": {
+      "get": {
+        "tags": [
+          "MetadataRoles"
+        ],
+        "description": "Lists roles for Purview Account",
+        "operationId": "MetadataRoles_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/RoleViewModelListResults"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Metadata Policy Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "MetadataRoles_List": {
+            "$ref": "./examples/MetadataRoles_List.json"
+          }
+        }
+      }
+    },
+    "/collections/{collectionName}/metadataPolicy": {
+      "get": {
```

Comment:
This path is odd since the API has no way to create a Collection or get/update/delete a specific collection by name.  I think this operation should be implemented as a filter on `get` of `/metadataPolicies` below.  Define a `collectionName` query parameter to pass in the collection name.

Reply:
The way to create a collection is using another Purview service
https://github.com/Azure/azure-rest-api-specs/blob/3257aacc47c2a03b7964cbb5c6a07ec9f2f232ee/specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2019-11-01-preview/account.json#L174

Reply:
We have  implemented this as a filter on get of /metadataPolicies now